source/includes/steps-create-index.yaml

```diff
@@ -1,21 +1,12 @@
-title: Click the :guilabel:`Create Index` button.
+title: Open the index creation dialog
```

Comment:
[note for review]

I adjusted the procedure for regular indexes to reflect the updated UI.